### PR TITLE
Feat: Deprecate markdown API documentation

### PR DESCRIPTION
**Pull request by Guaris**
Removes markdown API documentation from the 3.5 nav bar.
Adds redirects for markdown documentation pointing to API spec
Changes links in documentation to point to API documentation.

Re: https://github.com/Kong/docs.konghq.com/pull/6628

Preview: https://deploy-preview-6836--kongdocs.netlify.app/gateway/latest/admin-api/

### CODE REVIEW

**Comment by acgoldsmith** [comment]
@Guaris , can I suggest we utilize the same behavior as we do for linking to the specs from the Konnect nav, so they open in a new tab? 

```
        - text: API Spec
          url: /konnect/api/portal-management/latest/
          absolute_url: true
          generate: false
          target_blank: true
```

File: app/_data/docs_nav_gateway_3.5.x.yml [hunk 1]
```diff
@@ -520,61 +520,141 @@ items:
...
-      - text: Event Hooks
-        url: /admin-api/event-hooks/reference
-      - text: Keyring and Data Encryption
-        url: /admin-api/db-encryption
-      - text: Audit Logs
-        url: /admin-api/audit-logs
-      - text: Status API
-        url: /gateway/api/status/v1/
+        url: /admin-api/index_latest/
+      - text: Declarative Configuration
```
Comment (Guaris) [inline app/_data/docs_nav_gateway_3.5.x.yml:524]:
```suggestion
        url: /admin-api/index_latest/
        src: /admin-api/index
      - text: Declarative Configuration
```

File: app/_data/docs_nav_gateway_3.5.x.yml [hunk 2]
```diff
@@ -520,61 +520,142 @@ items:
...
-      - text: Event Hooks
-        url: /admin-api/event-hooks/reference
-      - text: Keyring and Data Encryption
-        url: /admin-api/db-encryption
-      - text: Audit Logs
-        url: /admin-api/audit-logs
-      - text: Status API
-        url: /gateway/api/status/v1/
+        url: /admin-api/index_latest/
+        src: /admin-api/index
```
Comment (lena-larionova) [inline app/_data/docs_nav_gateway_3.5.x.yml:524]:
```suggestion
        url: /admin-api/
        src: /gateway/admin-api/index_latest
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+### Documentation
+
+
+| Spec | Insomnia link |
+|-------|---------------|
+| [Enterprise beta API spec](/gateway/api/admin-ee/latest/) |<a href="https://insomnia.rest/run/?label=Kong%20Gateway%20Enterprise%203.4&uri=https%3A%2F%2Fraw.githubusercontent.com%2FKong%2Fdocs.konghq.com%2Fmain%2Fapi-specs%2FGateway-EE%2F3.4%2Fkong-ee-3.4.json" target="_blank"><img src="https://insomnia.rest/images/run.svg" alt="Run in Insomnia"></a>  |
+|  [Open source beta API spec](/gateway/api/admin-oss/latest/) |  <a href="https://insomnia.rest/run/?label=Kong%20Gateway%20Open%20Source%203.4&uri=https%3A%2F%2Fraw.githubusercontent.com%2FKong%2Fdocs.konghq.com%2Fmain%2Fapi-specs%2FGateway-OSS%2F3.4%2Fkong-oss-3.4.json" target="_blank"><img src="https://insomnia.rest/images/run.svg" alt="Run in Insomnia"></a>|
+
+
+|        |   Endpoints        | |
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:30]:
```suggestion
| Endpoints | | |
```
Moving the label to the first column. In context with the specs table above it, it looks odd to have the text centered:
Screenshot 2024-01-31 at 3 19 18 PM

Vs aligned to the left on both:
Screenshot 2024-01-31 at 3 19 11 PM

File: app/_src/gateway/admin-api/index_latest.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+ Requests to the Admin API can be sent to any node in the cluster, and Kong will
+ keep the configuration consistent across all nodes.
+
+ - `8001` is the default port on which the Admin API listens.
+ - `8444` is the default port for HTTPS traffic to the Admin API.
+
+ This API is designed for internal use and provides full control over Kong, so
+ care should be taken when setting up Kong environments to avoid undue public
+ exposure of this API. See [this document][secure-admin-api] for a discussion
+ of methods to secure the Admin API.
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:19]:
```suggestion
 exposure of this API. See [Securing the Admin API](/gateway/{{page.release}}/production/running-kong/secure-admin-api/)
 for more information about security methods.
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+### Documentation
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:21]:
```suggestion
## Documentation
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+| [CA Certificates](/gateway/api/admin-ee/latest/#/CA%20Certificates/list-ca_certificate) | [SNIs](/gateway/api/admin-ee/latest/#/SNIs/list-sni-with-certificate) | [Upstreams](/gateway/api/admin-ee/latest/#/Upstreams/list-upstream) |
+| [Targets](/gateway/api/admin-ee/latest/#/Targets/list-target-with-upstream) | [Vaults](/gateway/api/admin-ee/latest/#/Vaults/list-vault) | [Keys](/gateway/api/admin-ee/latest/#/Keys/list-key) |
+| [Filter Chains](/gateway/api/admin-ee/latest/#/filter-chains/get-filter-chains) | [Licenses](/gateway/api/admin-ee/latest/#/licenses/get-licenses) | [Workspaces](/gateway/api/admin-ee/latest/#/Workspaces/list-workspace) |
+| [RBAC](/gateway/api/admin-ee/latest/#/rbac/get-rbac-users) | [Admins](/gateway/api/admin-ee/latest/#/admins/get-admins) | [Consumer Groups](/gateway/api/admin-ee/latest/#/consumer_groups/) |
+| [Event Hooks](/gateway/api/admin-ee/latest/#/Event-hooks/get-event-hooks) | [Keyring and Data Encryption](/gateway/api/admin-ee/latest/#/Keyring/get-keyring) | [Audit Logs](/gateway/api/admin-ee/latest/#/audit-logs/get-audit-requests) |
+
+
+
+### DB-less Mode
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:43]:
```suggestion
## DB-less mode
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+
+
+In [DB-less mode](/gateway/{{page.kong_version}}/production/deployment-topologies/db-less-and-declarative-config/),
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:46]:
```suggestion
In [DB-less mode](/gateway/{{page.release}}/production/deployment-topologies/db-less-and-declarative-config/),
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+* [Validating plugin configurations against schemas](#validate-a-plugin-configuration-against-the-schema)
+* [Reloading the declarative configuration](#reload-declarative-configuration)
+* [Setting a target's health status in the load balancer](#set-target-as-healthy)
+
+
+
+
+
+
+
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:67]:
```suggestion
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+### Supported Content Types
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:68]:
```suggestion
## Supported content types
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+
+The Admin API accepts 3 content types on every endpoint:
+
+- **application/json**
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:72]:
```suggestion
### application/json
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+
+Handy for complex bodies (ex: complex plugin configuration), in that case simply send
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:74]:
```suggestion
The `application/json` content type is useful for complex bodies (for example, complex plugin configuration).
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+a JSON representation of the data you want to send. Example:
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:75]:
```suggestion
Send a JSON representation of the data you want to send. For example:
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+```json
+{
+    "config": {
+        "limit": 10,
+        "period": "seconds"
+    }
+}
+```
+
+An example adding a Route to a Service named `test-service`:
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:86]:
```suggestion
Here's an example of adding a route to a service named `test-service`:
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+
+```
+curl -i -X POST http://localhost:8001/services/test-service/routes \
+     -H "Content-Type: application/json" \
+     -d '{"name": "test-route", "paths": [ "/path/one", "/path/two" ]}'
+```
+
+- **application/x-www-form-urlencoded**
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:94]:
```suggestion
### application/x-www-form-urlencoded
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+
+Simple enough for basic request bodies, you will probably use it most of the time.
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:96]:
```suggestion
The content type `application/x-www-form-urlencoded` is useful for basic request bodies. 
You can use it in most situations.
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+Note that when sending nested values, Kong expects nested objects to be referenced
+with dotted keys. Example:
+
+```
+config.limit=10&config.period=seconds
+```
+
+When specifying arrays, send the values in order, or use square brackets (numbering
+inside the brackets is optional but if provided it must be 1-indexed, and
+consecutive). An example Route added to a Service named `test-service`:
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:106]:
```suggestion
consecutive). 

Here's an example route added to a service named `test-service`:
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 17]
```diff
@@ -0,0 +1,156 @@
...
+
+```
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:108]:
```suggestion
```sh
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 18]
```diff
@@ -0,0 +1,156 @@
...
+curl -i -X POST http://localhost:8001/services/test-service/routes \
+     -d "name=test-route" \
+     -d "paths[1]=/path/one" \
+     -d "paths[2]=/path/two"
+```
+
+The following two examples are identical to the one above, but less explicit:
+```
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:116]:
```suggestion
```sh
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 19]
```diff
@@ -0,0 +1,156 @@
...
+    -d "paths=/path/two"
+```
+
+
+- **multipart/form-data**
+
+Similar to URL-encoded, this content type uses dotted keys to reference nested
+objects. Here is an example of sending a Lua file to the pre-function Kong plugin:
+
+```
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:134]:
```suggestion
```sh
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 20]
```diff
@@ -0,0 +1,156 @@
...
+curl -i -X POST http://localhost:8001/services/plugin-testing/plugins \
+     -F "name=pre-function" \
+     -F "config.access=@custom-auth.lua"
+```
+
+When specifying arrays for this content-type, the array indices must be specified.
+An example Route added to a Service named `test-service`:
+
+```
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:143]:
```suggestion
```sh
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 21]
```diff
@@ -0,0 +1,156 @@
...
+     -d "paths[]=/path/two"
+
+curl -i -X POST http://localhost:8001/services/test-service/routes \
+    -d "name=test-route" \
+    -d "paths=/path/one" \
+    -d "paths=/path/two"
+```
+
+
+- **multipart/form-data**
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:129]:
```suggestion
### multipart/form-data
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 22]
```diff
@@ -0,0 +1,156 @@
...
+
+Similar to URL-encoded, this content type uses dotted keys to reference nested
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:131]:
```suggestion
The `multipart/form-data` content type is similar to URL-encoded. This content type uses dotted keys to reference nested
```

File: app/_src/gateway/admin-api/index_latest.md [hunk 23]
```diff
@@ -0,0 +1,156 @@
...
+objects. Here is an example of sending a Lua file to the pre-function Kong plugin:
+
+```
+curl -i -X POST http://localhost:8001/services/plugin-testing/plugins \
+     -F "name=pre-function" \
+     -F "config.access=@custom-auth.lua"
+```
+
+When specifying arrays for this content-type, the array indices must be specified.
+An example Route added to a Service named `test-service`:
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:141]:
```suggestion
For example, here's a route added to a service named `test-service`:
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]
Minor formatting edits, might as well get them out of the way.

File: app/_src/gateway/admin-api/index_latest.md [hunk 24]
```diff
@@ -0,0 +1,156 @@
...
+| [CA Certificates](/gateway/api/admin-ee/latest/#/CA%20Certificates/list-ca_certificate) | [SNIs](/gateway/api/admin-ee/latest/#/SNIs/list-sni-with-certificate) | [Upstreams](/gateway/api/admin-ee/latest/#/Upstreams/list-upstream) |
+| [Targets](/gateway/api/admin-ee/latest/#/Targets/list-target-with-upstream) | [Vaults](/gateway/api/admin-ee/latest/#/Vaults/list-vault) | [Keys](/gateway/api/admin-ee/latest/#/Keys/list-key) |
+| [Filter Chains](/gateway/api/admin-ee/latest/#/filter-chains/get-filter-chains) | [Licenses](/gateway/api/admin-ee/latest/#/licenses/get-licenses) | [Workspaces](/gateway/api/admin-ee/latest/#/Workspaces/list-workspace) |
+| [RBAC](/gateway/api/admin-ee/latest/#/rbac/get-rbac-users) | [Admins](/gateway/api/admin-ee/latest/#/admins/get-admins) | [Consumer Groups](/gateway/api/admin-ee/latest/#/consumer_groups/) |
+| [Event Hooks](/gateway/api/admin-ee/latest/#/Event-hooks/get-event-hooks) | [Keyring and Data Encryption](/gateway/api/admin-ee/latest/#/Keyring/get-keyring) | [Audit Logs](/gateway/api/admin-ee/latest/#/audit-logs/get-audit-requests) |
```
Comment (lena-larionova) [inline app/_src/gateway/admin-api/index_latest.md:39]:
```suggestion
The Kong Admin API is documented in OpenAPI format:

| Spec | Insomnia link |
|-------|---------------|
| [Enterprise beta API spec](/gateway/api/admin-ee/latest/) |<a href="https://insomnia.rest/run/?label=Kong%20Gateway%20Enterprise%203.4&uri=https%3A%2F%2Fraw.githubusercontent.com%2FKong%2Fdocs.konghq.com%2Fmain%2Fapi-specs%2FGateway-EE%2F3.4%2Fkong-ee-3.4.json" target="_blank"><img src="https://insomnia.rest/images/run.svg" alt="Run in Insomnia"></a>  |
|  [Open source beta API spec](/gateway/api/admin-oss/latest/) |  <a href="https://insomnia.rest/run/?label=Kong%20Gateway%20Open%20Source%203.4&uri=https%3A%2F%2Fraw.githubusercontent.com%2FKong%2Fdocs.konghq.com%2Fmain%2Fapi-specs%2FGateway-OSS%2F3.4%2Fkong-oss-3.4.json" target="_blank"><img src="https://insomnia.rest/images/run.svg" alt="Run in Insomnia"></a>|

See the following links for individual entity documentation:

{% navtabs %}
{% navtab Enterprise endpoints %}

| Endpoints    |           | |
| ------------------------------- | ------------------------------- | ----- | 
| [Information Routes](/gateway/api/admin-ee/latest/#/Information/get-endpoints) | [Health Routes](/gateway/api/admin-ee/latest/#/Information/get-status) | [Tags](/gateway/api/admin-ee/latest/#/tags/get-tags) |
| [Debug Routes](/gateway/api/admin-ee/latest/#/debug/put-debug-cluster-control-planes-nodes-log-level-log_level) | [Services](/gateway/api/admin-ee/latest/#/Services/list-service) | [Routes](/gateway/api/admin-ee/latest/#/Routes/list-route) |
| [Consumers](/gateway/api/admin-ee/latest/#/Consumers/list-consumer) | [Plugins](/gateway/api/admin-ee/latest/#/Plugins/list-plugins-with-consumer) | [Certificates](/gateway/api/admin-ee/latest/#/Certificates/list-certificate) |
| [CA Certificates](/gateway/api/admin-ee/latest/#/CA%20Certificates/list-ca_certificate) | [SNIs](/gateway/api/admin-ee/latest/#/SNIs/list-sni-with-certificate) | [Upstreams](/gateway/api/admin-ee/latest/#/Upstreams/list-upstream) |
| [Targets](/gateway/api/admin-ee/latest/#/Targets/list-target-with-upstream) | [Vaults](/gateway/api/admin-ee/latest/#/Vaults/list-vault) | [Keys](/gateway/api/admin-ee/latest/#/Keys/list-key) |
| [Filter Chains](/gateway/api/admin-ee/latest/#/filter-chains/get-filter-chains) | [Licenses](/gateway/api/admin-ee/latest/#/licenses/get-licenses) | [Workspaces](/gateway/api/admin-ee/latest/#/Workspaces/list-workspace) |
| [RBAC](/gateway/api/admin-ee/latest/#/rbac/get-rbac-users) | [Admins](/gateway/api/admin-ee/latest/#/admins/get-admins) | [Consumer Groups](/gateway/api/admin-ee/latest/#/consumer_groups/) |
| [Event Hooks](/gateway/api/admin-ee/latest/#/Event-hooks/get-event-hooks) | [Keyring and Data Encryption](/gateway/api/admin-ee/latest/#/Keyring/get-keyring) | [Audit Logs](/gateway/api/admin-ee/latest/#/audit-logs/get-audit-requests) |
{% endnavtab %}
{% navtab OSS endpoints %}
| Endpoints    |           | |
| ------------------------------- | ------------------------------- | ----- | 
| [Information Routes](/gateway/api/admin-oss/latest/#/Information/get-endpoints) | [Health Routes](/gateway/api/admin-oss/latest/#/Information/get-status) | [Tags](/gateway/api/admin-oss/latest/#/tags/get-tags) |
| [Debug Routes](/gateway/api/admin-oss/latest/#/debug/put-debug-cluster-control-planes-nodes-log-level-log_level) | [Services](/gateway/api/admin-oss/latest/#/Services/list-service) | [Routes](/gateway/api/admin-oss/latest/#/Routes/list-route) |
| [Consumers](/gateway/api/admin-oss/latest/#/Consumers/list-consumer) | [Plugins](/gateway/api/admin-oss/latest/#/Plugins/list-plugins-with-consumer) | [Certificates](/gateway/api/admin-oss/latest/#/Certificates/list-certificate) |
| [CA Certificates](/gateway/api/admin-oss/latest/#/CA%20Certificates/list-ca_certificate) | [SNIs](/gateway/api/admin-oss/latest/#/SNIs/list-sni-with-certificate) | [Upstreams](/gateway/api/admin-oss/latest/#/Upstreams/list-upstream) |
| [Targets](/gateway/api/admin-oss/latest/#/Targets/list-target-with-upstream) | [Vaults](/gateway/api/admin-oss/latest/#/Vaults/list-vault) | [Keys](/gateway/api/admin-oss/latest/#/Keys/list-key) |
{% endnavtab %}
{% endnavtabs %}
```
A few things happening here:

* Intro sentences for each table.
* We still need links to the OSS doc. How about tabs like this?
* Moving the `Endpoints` label to the first column to align the table. In context with the specs table above it, it looks odd to have the text centered:
Screenshot 2024-01-31 at 3 19 18 PM

Vs aligned to the left on both:
Screenshot 2024-01-31 at 3 19 11 PM

Comment (lena-larionova) [reply app/_src/gateway/admin-api/index_latest.md:39]: or, one more option: if we're adding tabs that already say "endpoints", how about removing the table header entirely: 
Screenshot 2024-01-31 at 3 46 58 PM

**Pull request merged**